markdown/plext/using_pljava.html.md.erb

```diff
@@ -2,205 +2,222 @@
-	- pljavau, UNTRUSTED PL/Java language
 - Transaction and Savepoint listeners enabling code execution when a transaction or savepoint is committed or rolled back.
 - Integration with GNU GCJ on selected platforms.
 
-A function in SQL will appoint a static method in a Java class. In order for the function to execute, the appointed class must available on the class path specified by the HAWQ server configuration parameter `pljava_classpath`. The PL/Java extension adds a set of functions that helps to install and maintain the Java classes. Classes are stored in normal Java archives, JAR files. A JAR file can optionally contain a deployment descriptor that in turn contains SQL commands to be executed when the JAR is deployed or undeployed. The functions are modeled after the standards proposed for SQL 2003.
-
-PL/Java implements a standard way of passing parameters and return values. Complex types and sets are passed using the standard JDBC ResultSet class.
-
-A JDBC driver is included in PL/Java. This driver calls HAWQ internal SPI routines. The driver is essential since it is common for functions to make calls back to the database to fetch data. When PL/Java functions fetch data, they must use the same transactional boundaries that are used by the main function that entered PL/Java execution context.
+PL/Java is optimized for performance. The Java virtual machine executes within the same process as the backend, minimizing call overhead. PL/Java brings the power of Java to the database itself, enabling the database-intensive business logic to execute as close to the actual data as possible.
 
-PL/Java is optimized for performance. The Java virtual machine executes within the same process as the backend to minimize call overhead. PL/Java is designed with the objective to enable the power of Java to the database itself so that database intensive business logic can execute as close to the actual data as possible.
-
-The standard Java Native Interface (JNI) is used when bridging calls between the backend and the Java VM.
+Calls between the HAWQ backend and the Java VM are bridged using the standard Java Native Interface (JNI).
 
 
 ## <a id="abouthawqpljava"></a>About HAWQ PL/Java 
 
+PL/Java is embedded in your HAWQ product distribution or within your HAWQ build if you chose to enable it as a build option. To use PL/Java in a HAWQ deployment, you must explicitly enable the PL/Java extension in all desired databases.
+
 There are a few key differences between the implementation of PL/Java in standard PostgreSQL and HAWQ.
 
-### <a id="pljavafunctions"></a>Functions 
+### <a id="pljavafunctions"></a>Functions Not Supported
 
-The following functions are not supported in HAWQ. The classpath is handled differently in a distributed HAWQ environment than in the PostgreSQL environment.
+The following functions are not supported in HAWQ. The PL/Java classpath is handled differently in a distributed HAWQ environment than that of the PostgreSQL environment.
```

Comment:
I think the sentence beginning "The PL/Java classpath..." Should be moved to afte this function list.